cylc/flow/scripts/remove.py

```diff
@@ -61,6 +83,19 @@ def get_option_parser() -> COP:
         argdoc=[FULL_ID_MULTI_ARG_DOC],
     )
 
+    parser.add_option(
+        '--flow',
+        action='append',
+        dest='flow',
+        metavar='FLOW',
+        help=(
+            "Remove the task(s) from the specified flow number. "
+            "Reuse the option to remove the task(s) from multiple flows. "
+            "If the option is not used at all, the task(s) will be removed "
+            "from all flows."
```

Comment:
```suggestion
            "By default, the task(s) will be removed "
            "from all flows."
```
(We use "by default" for this all over the place).